api/v1beta1/sparkapplication_types.go

```diff
@@ -17,149 +17,24 @@ limitations under the License.
 package v1beta1
 
 import (
-	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// SparkApplicationType describes the type of a Spark application.
-type SparkApplicationType string
-
-// Different types of Spark applications.
-const (
-	JavaApplicationType   SparkApplicationType = "Java"
-	ScalaApplicationType  SparkApplicationType = "Scala"
-	PythonApplicationType SparkApplicationType = "Python"
-	RApplicationType      SparkApplicationType = "R"
-)
-
-// DeployMode describes the type of deployment of a Spark application.
-type DeployMode string
-
-// Different types of deployments.
-const (
-	ClusterMode         DeployMode = "cluster"
-	ClientMode          DeployMode = "client"
-	InClusterClientMode DeployMode = "in-cluster-client"
-)
-
-// RestartPolicy is the policy of if and in which conditions the controller should restart a terminated application.
-// This completely defines actions to be taken on any kind of Failures during an application run.
-type RestartPolicy struct {
-	Type RestartPolicyType `json:"type,omitempty"`
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Same question here-- Who is this targeted at?

Reply:
The structure of the project is scaffolded by kubebuilder, so there are will be some comments to remind you to edit the file.